setup/win32/requirements-local-proxy.txt

```diff
@@ -1,3 +1,5 @@
 netifaces==0.11.0
 PyKCS11==1.5.10
 ghostscript==0.7
+cryptocode==0.1
+PySmbClient==0.1.5
```

Comment:
I'm not sure about what is the point here ?
smbclient on windows ?
samba is for unix/linux AFAIK